Common/ListExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Common
+{
+    public static class ListExtensions
+    {
+        public static List<T> AddRangeReturn<T>(this List<T> value, IEnumerable<T> collection)
+        {
+            value.AddRange(collection);
+            return value;
+        }
```

Comment:
that's not worth an extension method imo, especially since you can "flatten" lists of lists with SelectMany